hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/storage/row/HoodieRowDataParquetWriter.java

```diff
@@ -49,8 +49,7 @@ public HoodieRowDataParquetWriter(Path file, HoodieRowDataParquetConfig parquetC
     this.file = HoodieWrapperFileSystem.convertToHoodiePath(file, parquetConfig.getHadoopConf());
     this.fs = (HoodieWrapperFileSystem) this.file.getFileSystem(FSUtils.registerFileSystem(file,
         parquetConfig.getHadoopConf()));
-    this.maxFileSize = parquetConfig.getMaxFileSize()
-        + Math.round(parquetConfig.getMaxFileSize() * parquetConfig.getCompressionRatio());
+    this.maxFileSize = Math.round(parquetConfig.getMaxFileSize() * parquetConfig.getCompressionRatio());
     this.writeSupport = parquetConfig.getWriteSupport();
```

Comment:
The logic is referenced from `HoodieBaseParquetWriter`, why you think it is wrong, can you explain more details ?

Reply:
Hi Danny, I was referring to `HoodieAppendHandle`. I rechecked the code and found I understand it wrong. I'll close the PR, thanks!